gammapy/estimators/ts_map.py

```diff
@@ -146,20 +132,12 @@ def __init__(
         model=None,
         kernel_width="0.2 deg",
         downsampling_factor=None,
```

Comment:
I think we should also remove the downsampling option here soon and provide a more general solution that also works for other map estimators. However this can be done in a different PR...